packages/admithub-lightbox/lib/client/lightBox.js

```diff
@@ -0,0 +1,105 @@
+
+Meteor.startup(function(){
+  Meteor.setTimeout(function(){
+      if (Session.get('lightBoxPageViewSetting') === undefined ) {
+        Session.set('lightBoxPageViewSetting', checkShowSettings());
+      }
+  }, 60*1000 );
+});
+
+Telescope.modules.add("top", {
+template: "newsletter_alt",
+order: 0
+});
+
+var addMask = function() {
+ $('body').addClass('pg-mask');
+}
+
+var removeMask = function() {
+  $('body').removeClass('pg-mask');
+}
+
+var checkShowSettings = function () {
+  if (
+    !Users.can.view(Meteor.user())
+    ||  Session.get('lightBoxPageViewSetting') === false
+    ||  Cookie.get('showCustomBanner') === "no"        
+    ||  (Meteor.user() && Meteor.user().getSetting('profile.dontShowCustomEmailBanner', true) === true)
+    ) {
+      removeMask();
+      return false
+    } else {
+      addMask();
+      return true
+    }
+  }
+
+Template.layout.events({
+  'click a': function (e) {
+    Session.get('lightBoxPageViewCount') === undefined ? Session.set('lightBoxPageViewCount', 1 ) : Session.set('lightBoxPageViewCount', (Session.get('lightBoxPageViewCount') + 1 )) ;
+    if (Session.get('lightBoxPageViewCount') === 4 ) { Session.set('lightBoxPageViewSetting', checkShowSettings() ) }
```

Comment:
That's cute.  I never thought of setting the return value of a function as a session variable.